src/bundle/Controller/UserSettingsController.php

```diff
@@ -112,6 +112,19 @@ public function updateAction(Request $request, UpdateView $view)
                     ]);
                 }
 
+                if ($request->query->has('route')) {
```

Comment:
Where this `route` param is populated?

Also, I guess return url being part of form make more sense.

Reply:
https://github.com/ibexa/page-builder/pull/339/files#diff-8781e1f4fb2943dbbdf833e8a9e268ef9032bc749ad3bc25609bed0816b3e4cdR1379